src/components/layouts/landingPage/ArticuleSection/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import HeadingText from 'Atoms/HeadingText';
+import Image from 'Atoms/Image';
+import __fontSizes from 'FontSizes';
+import __devices from 'Devices';
+
+const ArticulesSection = () => {
```

Comment:
Change this to use `function declaration. Also, you made an error in the spelling of **Articles**
```suggestion
function ArticlesSection{
```

Reply:
Is there a reason for this